jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -654,6 +646,12 @@ else if (ExceptionUtil.areNotAssociated(cause, x))
         return !suspended;
     }
 
+    private void reopen()
```

Comment:
Because previously there was a `dispatch(Dispatchable)` method that once upon a time did lots of stuff but ultimately was reduced down to just doing a reopen and calling the dispatchable.... but I have inlined this method to get rid of the creation of the lambda dispatchable... but that means I need to call reopen every where there used to be a call to dispatch.   A small price to pay for less garbage on every dispatch.